src/js/components/tabs/tabs.links.spec.tsx

```diff
@@ -0,0 +1,619 @@
+import {TabsBehavior} from "./tabs";
+import {simulateKeyupEvent} from "../../../../jest-setup";
+
+describe('tab initialization behavior', () => {
+  it('should not throw an error when a selected tab button does not define what it controls', () => {
+    document.body.innerHTML = `
+        <div class="jazz-tablist" role="tablist">
+          <button role="tab">Tab Label 1</button>
```

Comment:
The tests for this file should've been testing for the a tags (anchor tags) right? Did you forget to edit this file by accident?

Reply:
Oh I thought adjusting the unit tests for the tabs was another ticket, and I was working on the tests now.

Should I add in the changes for the unit tests before merging this ticket?